manage.py

```diff
@@ -85,6 +85,7 @@ def sql_files(directory, tables_only=False):
             content = f.read()
         if tables_only:
             content = re.sub(r'^(CREATE|DROP) VIEW', r'\1 TABLE', content, flags=flags)
+        content = content.replace('EXTRAWHERECLAUSE', extra_where_clause)
```

Comment:
Once this is made into a comment as suggested, I think we can default `extra_where_clause`/`where_fragment` to `None` and then do this replacement `if where_fragment:`